project/BuildSettings.scala

```diff
@@ -0,0 +1,64 @@
+import sbt._
+import sbt.Keys._
+import sbtassembly.Plugin._
+import AssemblyKeys._
+
+import sbtbuildinfo.Plugin._
+
+
+
+import com.socrata.cloudbeessbt.SocrataCloudbeesSbt.{socrataBuildSettings, socrataProjectSettings}
+
+
+object BuildSettings {
+  val Organization = "com.socrata"
+
+  val buildSettings = socrataBuildSettings ++ buildInfoSettings ++
+    Seq(
+      name := "geospace",
+      scalaVersion := "2.10.3",
+      organization := Organization,
+      autoAPIMappings := true,
+      apiMappings += (
```

Comment:
What are these "example.org" autoAPIMappings and apiMappings settings used for? If we're not actually using them for anything real we should remove them.